Makefile

```diff
@@ -211,6 +211,16 @@ talos: buildkitd
 		$(COMMON_ARGS)
 	@docker load < build/$@.tar
 
+.PHONY: talos-cli
+talos-cli: buildkitd
+	@$(BINDIR)/buildctl --addr $(BUILDKIT_HOST) \
+		build \
+		--output type=docker,dest=build/$@.tar,name=docker.io/autonomy/$@:$(TAG) \
```

Comment:
I know right now we are namespace with `autonomy` but I plan on fixing that. I think I
'd prefer this to just be called `cli`. Then when we move to something like GH packages, or a registry we host, we can have the full image be something like `talos/cli:latest`.